src/main/webapp/javascript/user-auth.js

```diff
@@ -56,7 +62,7 @@ function signOut() {
  * @param {GoogleUser} current user to provide profile info.
  */
 function insertUserInfo(googleUser) {
-  // insert user name into user-page anchor
+  // Insert user name into user-page anchor
```

Comment:
Super nit: period at end of sentence

(mostly so we can try adding a new commit to see if it undoes approval)